Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -182,7 +182,7 @@ The configuration looks like this:
 
 :yaml:`routePath` defines the static keyword and the placeholders.
 :yaml:`requirements.category_id` is a unique integer of the category. 
-Without `_arguments` the parameter must have been 'category_id' as well .
+Without using `_arguments` the parameter the placeholder has to exactly match the argument name from URI which is `category`.
```

Comment:
Sorry to interrupt you again. There is something duplicated in your sentence. Do you prefer "the parameter" or "the placeholder"?